tests/tools/oesign/test-inputs/CMakeLists.txt

```
@@ -56,14 +56,32 @@ add_custom_command(
   COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/make-oesign-config.py
           --config_file new_security_version.conf --security_version 2)
 
+add_custom_command(
+  OUTPUT invalid_debug.conf
+  COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/make-oesign-config.py
+          --config_file invalid_debug.conf --debug 333)
+
 # Generate empty config file
 add_custom_command(
   OUTPUT empty.conf COMMAND cmake -E touch
                             ${CMAKE_CURRENT_BINARY_DIR}/empty.conf)
 
+add_custom_command(
+  OUTPUT invalid_syntax_1.conf
+  COMMAND cmake -E copy ${CMAKE_CURRENT_SOURCE_DIR}/invalid_syntax_1.conf
```

Per Anand's comments in review of #3128, should explicitly specify the source file dependency:
```suggestion
  DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/invalid_syntax_1.conf
  COMMAND cmake -E copy ${CMAKE_CURRENT_SOURCE_DIR}/invalid_syntax_1.conf
```